modules/installation-user-infra-machines-advanced.adoc

```diff
@@ -66,13 +66,13 @@ officially supported for making `/var` or a subdirectory of `/var` a separate
 partition, but not both. If you create more than one partition, Kubernetes
 will not be able to monitor them both.
 
-* Retain existing partitions: For a brownfield installation, where you are reinstalling on an existing {product-title} node and want to retain data partitions from your previous operating system, there are both boot arguments and options to `coreos-installer` that allow you to retain existing data partitions.
+* Retain existing partitions: For a brownfield installation, where you are reinstalling on an existing {product-title} node and want to retain data partitions from your previous operating system, there are both boot arguments and options to `coreos-installer` that allow you to retain existing data partitions on a disk.
```

Comment:
Yeah, `previous operating system` is doing a lot of work there; I agree this could be clarified.  The key point we want to make is that if the data partitions were originally created by installing RHCOS, rather than by installing another operating system, customers should continue to use their existing Ignition configs and should not use the partition-saving feature.

Reply:
Thanks, @miabbott and @bgilbert. This is helpful. I've revised this section to front-load Benjamin's key point and make it OS-agnostic in brownfield.